src/SigningPortal/SigningPortal.tsx

```diff
@@ -320,6 +388,7 @@ export const SigningPortal: React.FC = () => {
           <div className="pb-3">
             <div className="font-semibold pb-2">Extrinsic Info:</div>
             {tx ? (
+              <>
```

Comment:
```suggestion
              <React.Fragment>
```
Not a blocker. I think using `React.Fragment` makes the code easier to read, feel free to apply if you like it. I'm okay with both styles.

Reply:
Nice. https://github.com/r0gue-io/pop-wallet-signing-portal/pull/7/commits/76d80128bbf245c7e6f1184f6ed2649430a13a1f